components/_shared/SideNavigation.vue

```diff
@@ -51,93 +51,119 @@
       </nav>
 
       <footer class="SideNavigation-Footer">
-        <div class="SideNavigation-Social">
-          <app-link
-            to="https://line.me/R/ti/p/%40822sysfc"
-            :show-icon="false"
-            class="SideNavigation-SocialLink"
+        <div class="SideNavigation-Footer-Rows">
+          <ul class="SideNavigation-Social clearfix">
```

Comment:
```suggestion
          <ul class="SideNavigation-Social">
```

display:flexにすれば`clearfix` がなくても大丈夫だと思います。